bayes_opt.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+from scipy.optimize import minimize
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import Matern
+
+
+class BayesianOptimizer(object):
+
+    def __init__(self, feature_meta,
```

Comment:
Eh, nah I think sampling uniformly from the X's themselves makes the most sense, no real reason not too, eh?